jupyterlab/commands.py

```diff
@@ -554,7 +554,7 @@ def build(self, name=None, version=None, static_url=None,
                         'prod')
         command = ':'.join(parts)
 
-        self.logger.info(f'Building jupyterlab assets ({command})')
+        self.logger.info('Building jupyterlab assets (%s)'%command)
```

Comment:
```suggestion
        self.logger.info('Building jupyterlab assets (%s)' % command)
```

Reply:
Saved for a follow-up PR so we can merge without waiting for another test cycle and release today.

Reply:
https://gitter.im/jupyterlab/jupyterlab?at=5d68634ace6f764db2fee102